broker-node/app/Clients/iota-php-port/kerl-php/conv.php

```diff
@@ -58,8 +26,35 @@ function trytes_to_trits($trytes)
 
 function trits_to_trytes($trits)
 {
-    global $trit_table;
-    global $tryte_table;
+    $tryte_table = array(
```

Comment:
was NULL using the global var strategy above. we can figure out why post-testnet A